pkg/models/job.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package models
+
+import (
+	"time"
+
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/utils"
+)
+
+const JobTableName = "job"
+
+func NewJobId() string {
+	return utils.GetUuid("j-")
+}
+
+type Job struct {
+	JobId       string
+	ClusterId   string
+	AppId       string
+	AppVersion  string
+	JobAction   string
+	Directive   string
+	Owner       string
+	Status      string
+	ErrorCode   uint32
+	Executor    string
+	TaskCount   uint32
+	CreateTime  time.Time
+	StatusTime  time.Time
+}
+
+var JobColumns = GetColumnsFromStruct(&Job{})
+
+func NewJob(clusterId, appId, appVersion, jobAction, directive, userId string) *Job {
+	return &Job{
```

Comment:
It looks like this line of code is not formatted.

Reply:
Yes, it looks like `make build` not auto fmt code.